__fixtures__/test-project/web/package.json

```diff
@@ -27,6 +27,7 @@
     "postcss": "^8.4.23",
     "postcss-loader": "^7.3.0",
     "prettier-plugin-tailwindcss": "^0.2.8",
+    "storybook": "7.0.12",
```

Comment:
I need this here to get tests to run correctly; we can remove it after a canary gets published